### PR TITLE
Fixed issue on medium screens

### LINKED ISSUE #208: BUG: the medium screen view is not responsive
### There is an issue with responsiveness on the medium screen 
i could fix this also if am assigned to it

### Screenshots

image

### PR DESCRIPTION
**Pull request by dev-phantom**
Closes #208 

### What new changes did you make? Tick all applicable boxes
- [ ] Added new abbreviation
- [ ] Fixed something in the source code
- [x] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
I added responsiveness to the medium screen
added some hover to the tweet card 

### Share a screenshot of new change
localhost_5173_(iPad Air) (1)

### CODE REVIEW

**Comment by dev-phantom** [comment]
please i need ideas to make this better

**Comment by Njong392** [comment]
Will check it out and review later.

**Review by rupali-codes — COMMENTED** [review]
Good catch! but I got something -
*  `IYKYK` tags having too much gap in-between (see the space between `IYKYK` & `IF YOU KNOW, YOU KNOW`)
#### screenshot
image

* It would look more nice if we can flat-up second card too, just like the previous one
#### screenshot
image

**Comment by dev-phantom** [comment]
> Good catch! but I got something -
> 
> * `IYKYK` tags having too much gap in-between (see the space between `IYKYK` & `IF YOU KNOW, YOU KNOW`)
> 
> #### screenshot
> image
> 
> * It would look more nice if we can flat-up second card too, just like the previous one
> 
> #### screenshot
> image

thanks for catching these i'll work on it soon

**Comment by dev-phantom** [comment]
@rupali-codes you can check now i think this better

**Review by rupali-codes — COMMENTED** [review]
awesome! LGTM :)

File: src/components/hero.jsx [hunk 1]
```diff
@@ -101,8 +101,8 @@ const Hero = () => {
         </div>
       </section>
 
-      <section className=" justify-center gap-10 mt-10 items-center flex flex-col lg:flex-row">
-        <div className="border-2 p-4 rounded-lg border-purple md:rotate-3 relative md:mt-0 dark:border-deeppurple">
+      <section className=" justify-center lg:gap-10 mt-10 items-center flex flex-col lg:flex-row">
+        <div className="border-2 p-4 rounded-lg border-purple lg:rotate-3 hover:lg:rotate-0 hover:lg:ease-in-out hover:lg:duration-300 relative md:mt-0 dark:border-deeppurple">
```
Comment (Njong392) [inline src/components/hero.jsx:105]: Awesome animation you added here! I love it!

File: src/components/hero.jsx [hunk 2]
```diff
@@ -49,8 +49,8 @@ const Hero = () => {
 
       <Form />
 
-      <section className="block justify-center  items-center md:flex">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
+      <section className="block justify-center  items-center md:flex md:flex-col-reverse xl:flex xl:flex-row lg:flex lg:flex-row">
+        <div className="lg:w-1/2 xl:w-1/2 px-10 pt-5 lg:px-0 xl:px-0 lg:pr-20 lg:text-left md:text-left text-center">
```
Comment (Njong392) [inline src/components/hero.jsx:53]: I think there is a bit of redundancy here. What do you think?
```suggestion
        <div className="lg:w-1/2 xl:w-1/2 px-10 pt-5 lg:px-0 xl:px-0 lg:pr-20 md:text-left text-center">
```

File: src/components/form.jsx [hunk 3]
```diff
@@ -69,22 +69,24 @@ const Form = () => {
...
+      <section className="block justify-center md:pb-16 md:flex md:flex-col lg:flex xl:flex lg:flex-row xl:flex-row items-center">
+        <div className="md:w-full xl:pr-20 lg:pr-20 lg:w-1/2 xl:w-1/2 flex flex-col">
+          <h2 className="lg:text-left text-center text-purple font-bold text-xl md:text-2xl lg:text-3xl xl:text-3xl dark:text-deeppurple">
             <span className="text-ash dark:text-dark">
               Start by entering a slang,
             </span>{" "}
             and our dictionary will spit out an abbreviation.{" "}
           </h2>
-          <p className="text-gray text-left lg:text-center text-sm mt-5">
+          <p className="text-gray text-left xl:text-left lg:text-left md:text-center px-0 md:px-10 xmd:px-20 lg:px-0 xl:px-0 text-sm mt-5">
```
Comment (Njong392) [inline src/components/form.jsx:80]: Redundancy here too?
```suggestion
          <p className="text-gray text-left lg:text-left md:text-center px-0 md:px-10 xmd:px-20 lg:px-0 xl:px-0 text-sm mt-5">
```

File: src/components/form.jsx [hunk 4]
```diff
@@ -69,22 +69,24 @@ const Form = () => {
 
   return (
     <div className="bg-dark py-5 mb-12 md:mb-0 lg:py-12 px-[14px] dark:bg-ash">
-      <section className="block justify-center md:pb-16 md:flex items-center">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
-          <h2 className="text-purple font-bold text-xl lg:text-3xl dark:text-deeppurple">
+      <section className="block justify-center md:pb-16 md:flex md:flex-col lg:flex xl:flex lg:flex-row xl:flex-row items-center">
+        <div className="md:w-full xl:pr-20 lg:pr-20 lg:w-1/2 xl:w-1/2 flex flex-col">
+          <h2 className="lg:text-left text-center text-purple font-bold text-xl md:text-2xl lg:text-3xl xl:text-3xl dark:text-deeppurple">
```
Comment (Njong392) [inline src/components/form.jsx:74]:
```suggestion
          <h2 className="lg:text-left text-center text-purple font-bold text-xl md:text-2xl lg:text-3xl dark:text-deeppurple">
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
This is are amazing changes that you have made! Thank you 🎉 But it also seems that you committed changes to the `package.json` and `package-lock.json`. These may cause changes to the main branch later if merged. lease check out [this guide](https://gist.github.com/kumarsonsoff3/fbc6ecde519250a9f0a4d7ccc9993a1a) to revert.

I really love this pull request! Great work, thank you.

File: src/components/hero.jsx (hunk 2 above)
Comment (dev-phantom) [reply src/components/hero.jsx:53]: I'll work on it

**Pull request closed without merging**